.eslintrc.json

```diff
@@ -1,11 +1,13 @@
 {
     "extends": "next/core-web-vitals",
```

Comment:
i wonder if since next is complaining certain files don't exist, maybe this is why? i wonder what happens if we switch to some other base config we wouldn't have to worry about it

[some discussion here](https://github.com/vercel/next.js/discussions/58714#discussioncomment-8324062)

Reply:
good catch! I got the error re: `next` when running `next lint` (removed [here](https://github.com/safeinsights/setup-app/pull/12/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L10)) -- but not when running `eslint`, even with this setting. but i agree this isn't the best plugin for us to use.

I tried removing the `extends` field as well as replacing the above line with `"extends": "eslint:recommended"`. Everything seems to work the same, except weirdly, I'm getting the below error in `sorter.js`

<img width="615" alt="image" src="https://github.com/user-attachments/assets/c6d65d1e-66d6-48a3-b0d4-28ef440859eb" />

Note that `eslint` is even disabled above 😱 . Going to keep looking into this ...

Reply:
woah 😮 that is so weird!!

Reply:
heads up that nextjs 15 upgrades eslint to the current version and it's config file format changes.  management app change to that is here:  https://github.com/safeinsights/management-app/pull/48/files#diff-9601a8f6c734c2001be34a2361f76946d19a39a709b5e8c624a2a5a0aade05f2

Reply:
okay, i removed `next/core-web-vitals` & fixed the above issue by adding config to `eslintrc` https://github.com/safeinsights/setup-app/pull/13/commits/311d2ecef866a31be6f1821fe5db4320a79a2de7

then the CLI broke in a new weird way (only on Github Actions, worked locally fine). so I updated eslint per nathan's recommendation & migrated `eslintrc` to `eslint.config.mjs` using the auto-migrator. now everything works again 🤞... 

so that was cool and easy and everything worked as expected

Reply:
i mean -- the config SHOULD be fine for this project. i just feel like some of these other buggy configs should also have been fine. but whatever 🤷